src/kubernetes/hash_value.rs

```diff
@@ -34,6 +44,12 @@ where
         }
         Some(metadata.uid.as_ref()?.as_str())
     }
+
+    /// Get the `namespace` from the `T`'s [`Metadata`] (if any).
+    pub fn name(&self) -> Option<&str> {
+        let metadata = self.0.metadata();
+        Some(metadata.name.as_ref()?.as_str())
```

Comment:
I realize `uid()` is already doing this too, but it seems like we always return `Some()` here? Is there a reason this needs to return an `Option`?

Reply:
Nvm, I missed the `?` next to `as_ref`.

Reply:
Rust doesn't like your suggestion - how important is this to you, since it's mimicking the existing method here for uid

Reply:
I'm curious what it is complaining about, but yeah, definitely not a big deal.